src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,877 @@
+	        FrameUtilities.showMessage(this, "Trying to use an invalid CDP?", warningMessage);
+	    }
+
+        List<File> allFiles = ListUtilities.getAllJudgeSampleSubmissionFilenamesFromCDP(getContest(), cdpTextField.getText());
+
+        if (allFiles.size() == 0) {
+            FrameUtilities.showMessage(this, "No Runs", "No Runs found under " + cdpTextField.getText());
+            return;
+        }
+
+        boolean submitYesSamples = checkBoxSubmitYesSamples.isSelected();
+        boolean submitNoSamples = checkBoxSubmitFailingSamples.isSelected();
+
+        if (!submitYesSamples && !submitNoSamples) {
+            FrameUtilities.showMessage(this, "No Runs", "Select either Yes or Failed runs");
+            return;
+        }
+
+        List<File> files = new ArrayList<File>();
+
+        files = QuickSubmitter.filterRuns(allFiles, submitYesSamples, submitNoSamples);
+
+        if (submitNoSamples && submitSelectedJudgingTypeRadioButton.isSelected()) {
+            if (submitSelectedJudgingTypeRadioButton.isSelected()) {
+                // if they selected the radio button for selected, they should select at least
+                if (selectedJudgingTypeIndexes == null || selectedJudgingTypeIndexes.length == 0) {
+                    FrameUtilities.showMessage(this, "No Judging Types selected", "No Judging Types selected, select at least one");
+                    return;
+                }
+
+                // fitler by judging type (accepted, wrong_answer, etc.)
+                files = ListUtilities.filterByJudgingTypes(files, submissionSolutionList);
+            }
+        }
+	    
+        if (checkBoxProblems.isSelected() ) {
+            if (submitSelectedProblemsRadioButton.isSelected()) {
+                // submissions may be filtered by problem
+
+                if (selectedProblemsIndexes == null || selectedProblemsIndexes.length == 0) {
+                    FrameUtilities.showMessage(this, "No Problem selected", "No Problems selected, select at least one");
+                    return;
+                }
+
+                // filter list by problem name
+                files = ListUtilities.filterByProblems (files, selectedProblemList);
+
+            }
+        }
+	    
+	    if (checkBoxLanguage.isSelected()) {
+	        // submissions may be filtered by language
+	        
+	        if (submitSelectedLanguagesRadioButton.isSelected()) {
+	            if (selectedLanguagesIndexes == null || selectedLanguagesIndexes.length == 0) {
+	                FrameUtilities.showMessage(this, "No Language selected", "No Languages selected, select at least one");
+	                return;
+
+	            }
+
+	            files = ListUtilities.filterByLanguages(files, getContest(), selectedLanguageList);
+	        }
+	    }
+	    
+
+	    int count = 0;
+	    for (File file : files) {
+	        count++;
+	        xlog("Will submit #" + count + " file =" + file.getAbsolutePath());
```

Comment:
This log message is somewhat confusing -- it shows up on the log text area (once for each separate file, of which there could be hundreds), saying "will submit".  In fact however, the determination of whether the files _get submitted_ is made when the user clicks the `Submit` button (and then responds to the Yes/No/Cancel on that dialog).
I think it would be much clearer if each log message was (something like) 
```
File selected for submission: (1) c:\home\myfile1
```
It indicates that the file has been _selected_ for submission (instead of saying it "_will be submitted_".  Minor subtle point.... but we should do what we can to help users understand the GUI...